### PR TITLE
feat(security-apps): add Open Policy Agent

### PR DESCRIPTION
**Pull request by hairmare**

### CODE REVIEW

File: charts/security-apps/examples/opa.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+falco:
```
Comment (eyenx) [inline charts/security-apps/examples/opa.yaml:1]:
```
opa:
```

File: charts/security-apps/README.md [hunk 2]
```diff
@@ -43,6 +43,13 @@ This chart is maintained by [Adfinis](https://adfinis.com/?pk_campaign=github&pk
 | gangway.repoURL | string | [repo](https://charts.helm.sh/stable/) | Repo URL |
 | gangway.targetRevision | string | `"0.4.*"` | [gangway Helm chart](https://github.com/helm/charts/tree/master/stable/gangway/) version |
 | gangway.values | object | [upstream values](https://github.com/helm/charts/tree/master/stable/gangway/values.yaml) | Helm values |
+| opa | object | - | [Open Policy Agent](https://www.openpolicyagent.org/) ([example](./examples/opa.yaml)) |
+| opa.chart | string | `"opa"` | Chart |
+| opa.destination.namespace | string | `"infra-opa"` | Namespace |
+| opa.enabled | bool | `false` | Enable Open Policy Agent |
+| opa.repoURL | string | [repo](https://charts.helm.sh/stable) | Repo URL |
```
Comment (eyenx) [inline charts/security-apps/README.md:50]: Is there really no official chart in https://github.com/open-policy-agent ?

Or can not this be done with gatekeeper?

**Review by eyenx — CHANGES_REQUESTED** [review]

**Comment by tongpu** [comment]
I guess we should rather be using gatekeeper: https://github.com/open-policy-agent/gatekeeper#deploying-via-helm
And before we do that we should figure out rego. ;)

**Comment by hairmare** [comment]
This should indeed be gatekeeper and not stable/opa I'll make a new pr with gatekeeper.

**Pull request closed without merging**